Mikado/Mikado/Model/TripCell.swift

```diff
@@ -5,30 +5,53 @@
 //  Created by Aziz Zaynutdinov on 2/10/19.
 //  Copyright © 2019 Aziz Zaynutdinov. All rights reserved.
 //
-
 import UIKit
 
 class TripCell: UITableViewCell {
 
+    @IBOutlet weak var cityImage: UIImageView!
+    @IBOutlet weak var tripName: UILabel!
+    
     override func awakeFromNib() {
         super.awakeFromNib()
         // Initialization code
+        self.layer.borderWidth = 1
+        self.layer.cornerRadius = 4
+        self.clipsToBounds = true
     }
 
     override func setSelected(_ selected: Bool, animated: Bool) {
         super.setSelected(selected, animated: animated)
 
         // Configure the view for the selected state
     }
-    
+
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
-        
-        backgroundColor = .yellow
     }
     
     required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+        super.init(coder: aDecoder)
+        //fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setLabel (_ name: String) {
+        tripName.text = name
+        tripName.textColor = .white
+        tripName.font = UIFont.systemFont(ofSize: 20.0)
+    }
+    
+    func setImage (_ im : UIImage) {
+        // set image
+        cityImage.image = im
+        cityImage.layer.opacity = 0.9
```

Comment:
You can set this once in init for lines 47-54 and remove this setImage function. Same with setLabel, you can set the textColor and font once and remove the setLabel function.

In tableview:cellForRow:, you can just do cell.cityImageView.image = UIImage named ..., same with tripName.